ebs/ebs.go

```diff
@@ -766,6 +779,10 @@ func (d *Driver) GetBackupInfo(backupURL string) (map[string]string, error) {
 		"State":         *ebsSnapshot.State,
 	}
 
+	if *ebsSnapshot.KmsKeyId != "" {
```

Comment:
It's OK to just put it in initial assignment. I am not worry about the extra empty key.